README.md

```diff
@@ -438,7 +438,9 @@ ANSWERS.addComponent('SearchBar', {
     enabled: false,
     // Optional, the message in the alert. Defaults to the below
     message: "We are unable to determine your location"
-  }
+  },
+  // Optional, a callback function that runs after the searchbar completes a search.
```

Comment:
What does `completes` a search mean here? Can we add that to the README?

Reply:
We run the `search` function when we do a query on load. Do we want to run `onSearchComplete` in that case? This is true generally whenever the QUERY is updated in globalStorage because of the `update` listener.

Reply:
oh i forgot about this beauty, even the searchbar relies on this query update listener (at least sometimes) it looks like, and debounced search makes things even uglier, I'll just do this a different way

Reply:
where is my cry react github? 😢

Reply:
It's a true cry react :_(